docs/plugins.rst

```diff
@@ -166,6 +166,9 @@ plugin under consideration.  The objects themselves are relatively simple:
       enabled: false
       path: path/to/quux
 
+    - name: docs
+      enabled: true
+
```

Comment:
Actually that is a pretty good idea.  The only reason *not* to that I can think of is that there's no associated command line argument to name a plugin for inclusion (all the other config file options have such an associated command line argument).  I don't think that's really enough reason not to merge these two things together, especially since if I really wanted to I could include an appropriate command line arg.